### PR TITLE
updated appointment and vital signs files

### PR DESCRIPTION
**Pull request by adieddygit**
Updated the Appointment and Vital signs files

### CODE REVIEW

File: appoint_register.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+  require_once "nav.php";
+ ?>
+ 
+<body>
+<div class="a_appoint">
+   <h1>Appointment Registration Form</h1>
+       <nav>
+           <a class="nav-link" href="#">Record ID</a>
+           <a class="nav-link" href="#">210775</a>
```
Comment (midyayee) [inline appoint_register.php]: Why is this not a readable name but a number?

File: css/appointment.css
Comment (midyayee) [inline css/appointment.css:1]: This is good but when building a new app. Alway consider making your CSS take a mobile first approach. Great Job. But you can rewatch part 2 to get a depth of the breakpoints

File: css/appointment.css [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+  .col-11 {
+    width: 91.66%;
+  }
+  
+  .col-12 {
+    width: 100%;
+  }
+  }
+  
+  /* Small devices (portrait tablets and large phones, 600px and up)*/
```
Comment (midyayee) [inline css/appointment.css]: Try putting a condition here for when e.g (min-width:600px and max-width : 768px)to ensure that you are not overlapping breakpoints. Research this to see the best approach.

File: css/vitalsigns.css
Comment (midyayee) [inline css/vitalsigns.css:1]: Why not make universal styles on a common css file and only put styles specific to a page on a new style sheet. For instance all your grid view and general responsiveness startup styles should have been on a css file to be included in all your pages and then page specific style on their individual style sheet and include but external files. You want to eliminate duplicate code as much as possible.

**Review by midyayee — CHANGES_REQUESTED** [review]
Consider the comment from each section and update your code as much as possible to meet the standards. For areas where the code cannot be updated, justify with reasons as a response to the comments.

**Comment by adieddygit** [comment]
Good afternoon, please have made some updates on the requested change in
previous commits

**Review by midyayee — APPROVED** [review]

**Pull request merged**